File: admin/src/api/interface/sysLoggerDev.ts
// sys_logger_dev 开发日志
export interface ReqSysLoggerDevParams extends RequestPage {
  beginTimestamp?: string // 时间
  finishTimestamp?: string // 时间
  host?: string // 服务名
  level?: string // 等级
}

export interface ResSysLoggerDev {
  id: number // 编号
  host: string | undefined // 服务名
  timestamp: string // 时间
  file: string | undefined //
  func: string | undefined // 方法名
  message: string | undefined // 消息
  level: string | undefined // 等级
  data: any | undefined // 数据
}

File: cloud/dao/sys_logger_dev.go
package dao

import "github.com/abulo/ratel/v3/stores/null"

// SysLoggerDev 开发日志 sys_logger_dev
type SysLoggerDev struct {
	Id        *int64        `gorm:"primaryKey;autoIncrement;column:id" json:"id"` //bigint 编号,PRI
	Host      null.String   `gorm:"column:host" json:"host"`                      //varchar 服务名
	Timestamp null.TimeStamp `gorm:"column:timestamp" json:"timestamp"`            //datetime 时间
	File      null.String   `gorm:"column:file" json:"file"`                      //varchar
	Func      null.String   `gorm:"column:func" json:"func"`                      //varchar 方法名
	Message   null.String   `gorm:"column:message" json:"message"`                //varchar 消息
	Level     null.String   `gorm:"column:level" json:"level"`                    //varchar 等级
	Data      null.JSON     `gorm:"column:data" json:"data"`                      //json 数据
}

func (SysLoggerDev) TableName() string {
	return "sys_logger_dev"
}
